src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/r/RCompletionToolTip.java

```diff
@@ -54,13 +57,55 @@ public void resolvePositionAndShow(String signature,
             rectangle.getTop());
    }
    
+   public void resolvePositionAndShow(String signature)
+   {
+      setCursorAnchor();
+      resolvePositionAndShow(signature, docDisplay_.getCursorBounds());
+   }
+   
+   private String truncateSignature(String signature)
```

Comment:
Right, I decided to go with the smaller display as e.g. functions like `read.table` have signatures way too large.